src/WCSimLIGen.cc

```diff
@@ -46,49 +46,76 @@ void WCSimLIGen::SetPhotonMode(G4bool photonmode){
 }
     
 
-void WCSimLIGen::ReadFromDatabase(G4String injectorType, G4String injectorIdx, G4String injectorFilename){
+void WCSimLIGen::ReadFromDatabase(G4String injectorType, G4String injectorIdx, G4String injectorFilename, G4String injectorDetails, G4String injectorDetector){
 
     // Define the database to read from
     string db = injectorFilename;
+    string db_pos = injectorDetails;
+    
     if (db.empty()) {
         db = wcsimdir + "LightInjectors.json";
     }
     else {
         db = wcsimdir + injectorFilename;
     }
+
+    if (db_pos.empty()) {
+        db_pos = wcsimdir + "LightInjectorsLocation.json";
```

Comment:
```suggestion
        db_pos = wcsimdir + "LightInjectorsDetails.json";
```
I think this is the correct default?